code/game/objects/items/weapons/bio_chips/bio_chip_adrenalin.dm

```diff
@@ -32,3 +32,35 @@
 	name = "bio-chip case - 'Adrenaline'"
 	desc = "A glass case containing an adrenaline bio-chip."
 	implant_type = /obj/item/bio_chip/adrenalin
+
+/obj/item/bio_chip/proto_adrenalin
+	name = "proto-adrenal bio-chip"
+	desc = "Removes all stuns and knockdowns."
```

Comment:
Maybe a different description? Or is there a reason to keep the same one as adrenals?

Reply:
oops!

Reply:
Well, looking at it again. Not much more could be added, is what is read to the action button, apart from mentioning that it is less effective, but unsure if that is needed on the action button